src/reducers/Settings.js

```diff
@@ -12,8 +12,8 @@ import {
 } from '../actions/Settings'
 
 const initialSettings = {
-  insightAPI: 'https://explorer.zensystem.io/insight-api-zen/',
-  explorerURL: 'https://explorer.zensystem.io/',
+  insightAPI: 'https://explorer.horizen.global/api/',
+  explorerURL: 'https://explorer.horizen.global/',
```

Comment:
I can see this URL dotted around in different places as a hard string, it would be much better moved to a constant like `LANG_ENGLISH` and `CURRENCY_USD` so we only need to update it in one place if it changes again